### PR TITLE
III-3551 Test for PHP 7.x and PHP 8.x compatilibility

### PR DESCRIPTION
**Pull request by LucWollants**
### Added
- Added Github pull request template

---
Ticket: https://jira.uitdatabank.be/browse/III-3551

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -4,19 +4,20 @@
   "license": "GPL-3.0",
   "authors": [
     {
-      "name": "Kristof Coomans",
-      "email": "kristof@2dotstwice.be"
+      "name": "Publiq vwz",
+      "email": "info@publiq.be"
```
Comment (bertramakers) [inline composer.json:8]: Good idea 👍 Let's do this for every repo we update in this ticket.

File: composer.json [hunk 2]
```diff
@@ -4,19 +4,20 @@
...
   "prefer-stable": true,
   "require": {
+    "php": ">=7.1",
     "cultuurnet/deserializer": "~0.1",
     "cultuurnet/valueobjects": "~3.0",
     "broadway/broadway": "~0.10"
   },
   "require-dev": {
-    "phpunit/phpunit": "~5.7",
+    "phpunit/phpunit": "^7.5",
```
Comment (bertramakers) [inline composer.json:20]: Nice that it was a quick fix 👍 
I'm wondering if the `satooshi/php-coveralls` dependency is still relevant? But that's definitely a nit and easy to remove later if needed.

**Review by bertramakers — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (bertramakers) [reply composer.json:8]: Ah there's a typo though. `vwz` should be `vzw`

File: composer.json [hunk 3]
```diff
@@ -4,19 +4,20 @@
   "license": "GPL-3.0",
   "authors": [
     {
-      "name": "Kristof Coomans",
-      "email": "kristof@2dotstwice.be"
+      "name": "Publiq vwz",
```
Comment (bertramakers) [inline composer.json:7]:
```suggestion
      "name": "Publiq vzw",
```

File: composer.json (hunk 1 above)
Comment (bertramakers) [reply composer.json:8]: I've added a suggested change below

File: composer.json (hunk 3 above)
Comment (bertramakers) [reply composer.json:7]: It seems like this typo is also in other repo's. Can you double-check the ones that you go over for this ticket?

File: composer.json (hunk 2 above)
Comment (jdrieghe) [reply composer.json:20]: I see a lot more value in introducing something like https://github.com/infection/infection over simple PHP coverage enforcement

File: composer.json (hunk 1 above)
Comment (LucWollants) [reply composer.json:8]: Good point!

File: composer.json (hunk 2 above)
Comment (LucWollants) [reply composer.json:20]: For now, I just removed the `satooshi/php-coveralls` dependency. Which other quality tools we want to support we can decide later.

**Pull request merged**